scripts/Dockerfile.test

```diff
@@ -29,6 +29,7 @@ RUN source /build.env \
 	librbd-devel \
 	rubygems \
 	ShellCheck \
+   codespell \
```

Comment:
correct indent

Reply:
Dont know, but it's fine in my local file